standalone/server.c

```diff
@@ -998,14 +998,26 @@ AP_DECLARE(void) unixd_pre_config(apr_pool_t *ptemp)
     apr_finfo_t wrapper;
 
 #if AP_SERVER_MAJORVERSION_NUMBER > 1 && AP_SERVER_MINORVERSION_NUMBER < 3
+#ifdef WAF_JSON_LOGGING_ENABLE
+    unixd_config.user_name = msc_waf_lock_owner;
+    unixd_config.user_id = ap_uname2id(msc_waf_lock_owner);
+    unixd_config.group_id = ap_gname2id(msc_waf_lock_owner);
+#else
     unixd_config.user_name = DEFAULT_USER;
```

Comment:
Is there a way to set DEFAULT_USER to msc_waf_lock_owner?

Reply:
yes, There is some way to do that. i think it might be more clean under WAF_JSON_LOGGING_ENABLE, we control everything for the permission, and this will make code looks cleaner. and we don't want to overwrite some pre defined maros.

Reply:
I didnt understand your response. Lets talk when you are in.